### ISSUE TITLE
[Documentation]: Rationale of To test if the CNF uses local storage is unclear

### ISSUE THREAD

**Issue by CsatariGergely** [cncf/cnf-testsuite#1340]
**Describe the bug**
Rationale of [To test if the CNF uses local storage](https://github.com/cncf/cnf-testsuite/blob/main/RATIONALE.md#to-test-if-the-cnf-uses-local-storage-no_local_volume_configuration) is not a rationale. It just says that a CNF should not do something, rationale should explain why the CNF should not do the thing. 

**Expected behavior**
Add a rationale.

**Comment by kosstennbl** [comment]
Agree, rationale of "no_local_volume_configuration" doesn't explain anything, should be updated.
